2018-05-10 09:11:01  config.py[line:24] INFO 数据库初始化完成
2018-05-10 09:11:02  _internal.py[line:88] INFO  * Running on http://0.0.0.0:9090/ (Press CTRL+C to quit)
2018-05-10 09:11:49  config.py[line:24] INFO *************************收到访问请求: /*************************
2018-05-10 09:11:49  config.py[line:24] INFO 数据总量：20557
2018-05-10 09:11:49  config.py[line:24] INFO *************************请求处理完毕*************************
2018-05-10 09:11:49  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 09:11:49] "GET / HTTP/1.1" 200 -
2018-05-10 09:11:50  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 09:11:50] "GET /htmls/common.js HTTP/1.1" 304 -
2018-05-10 09:11:50  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 09:11:50] "GET /htmls/jquery-3.3.1.js HTTP/1.1" 304 -
2018-05-10 09:11:50  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 09:11:50] "GET /htmls/common.css HTTP/1.1" 304 -
2018-05-10 09:11:50  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 09:11:50] "GET /favicon.ico HTTP/1.1" 404 -
2018-05-10 09:11:52  config.py[line:24] INFO *************************收到访问请求: /result*************************
2018-05-10 09:11:52  config.py[line:24] INFO 数据库分页查询sql: 

            SELECT * FROM items ORDER BY good_id LIMIT 20 OFFSET 0 * 20  where status <> "000" and  1=1 
        
2018-05-10 09:11:52  app.py[line:1761] ERROR Exception on /result [GET]
Traceback (most recent call last):
  File "D:\python3\lib\site-packages\flask\app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\python3\lib\site-packages\flask\app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\python3\lib\site-packages\flask\app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "D:\python3\lib\site-packages\flask\_compat.py", line 35, in reraise
    raise value
  File "D:\python3\lib\site-packages\flask\app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\python3\lib\site-packages\flask\app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "D:/py_ws/PythonNote/PythonNote/webService/src/flask_main.py", line 64, in result
    data = DBHelper.select_items(page=1, page_size=config['page_size'], options=('status <> "000"',))
  File "D:\py_ws\PythonNote\PythonNote\webService\src\db\dbhelper.py", line 148, in select_items
    return [GoodItem(info_tuple=item) for item in cls.cursor.execute(select_sql)]
sqlite3.OperationalError: near "where": syntax error
2018-05-10 09:11:53  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 09:11:53] "GET /result HTTP/1.1" 500 -
2018-05-10 09:22:29  config.py[line:24] INFO 数据库初始化完成
2018-05-10 09:22:30  _internal.py[line:88] INFO  * Running on http://0.0.0.0:9090/ (Press CTRL+C to quit)
2018-05-10 09:22:36  config.py[line:24] INFO *************************收到访问请求: /*************************
2018-05-10 09:22:36  config.py[line:24] INFO 数据总量：20557
2018-05-10 09:22:36  config.py[line:24] INFO *************************请求处理完毕*************************
2018-05-10 09:22:36  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 09:22:36] "GET / HTTP/1.1" 200 -
2018-05-10 09:22:37  config.py[line:24] INFO *************************收到访问请求: /items/1*************************
2018-05-10 09:22:37  config.py[line:24] INFO 数据库分页查询sql: 

            SELECT * FROM items  ORDER BY good_id LIMIT 20 OFFSET 0 * 20
        
2018-05-10 09:22:37  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 09:22:37] "GET /items/1 HTTP/1.1" 200 -
2018-05-10 09:22:38  config.py[line:24] INFO *************************收到访问请求: /result*************************
2018-05-10 09:22:38  config.py[line:24] INFO 数据库分页查询sql: 

            SELECT * FROM items  where status <> "000" and  1=1  ORDER BY good_id LIMIT 20 OFFSET 0 * 20
        
2018-05-10 09:22:39  config.py[line:24] INFO 数据总量：1
2018-05-10 09:22:39  config.py[line:24] INFO *************************请求处理完毕*************************
2018-05-10 09:22:39  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 09:22:39] "GET /result HTTP/1.1" 200 -
2018-05-10 09:37:00  config.py[line:24] INFO 数据库初始化完成
2018-05-10 09:37:00  _internal.py[line:88] INFO  * Running on http://0.0.0.0:9090/ (Press CTRL+C to quit)
2018-05-10 09:37:04  config.py[line:24] INFO *************************收到访问请求: /*************************
2018-05-10 09:37:04  config.py[line:24] INFO 数据总量：20557
2018-05-10 09:37:04  config.py[line:24] INFO *************************请求处理完毕*************************
2018-05-10 09:37:04  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 09:37:04] "GET / HTTP/1.1" 200 -
2018-05-10 09:37:05  config.py[line:24] INFO *************************收到访问请求: /items/1*************************
2018-05-10 09:37:05  config.py[line:24] INFO 数据库分页查询sql: 

            SELECT * FROM items  ORDER BY good_id LIMIT 20 OFFSET 0 * 20
        
2018-05-10 09:37:05  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 09:37:05] "GET /items/1 HTTP/1.1" 200 -
2018-05-10 09:37:08  config.py[line:24] INFO *************************收到访问请求: /result*************************
2018-05-10 09:37:08  app.py[line:1761] ERROR Exception on /result [GET]
Traceback (most recent call last):
  File "D:\python3\lib\site-packages\flask\app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\python3\lib\site-packages\flask\app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\python3\lib\site-packages\flask\app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "D:\python3\lib\site-packages\flask\_compat.py", line 35, in reraise
    raise value
  File "D:\python3\lib\site-packages\flask\app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\python3\lib\site-packages\flask\app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "D:/py_ws/PythonNote/PythonNote/webService/src/flask_main.py", line 64, in result
    count_has_front = DBHelper.select_count(options=('status likes "1%"',))
  File "D:\py_ws\PythonNote\PythonNote\webService\src\db\dbhelper.py", line 162, in select_count
    res = cls.cursor.execute(select_sql)
sqlite3.OperationalError: near "likes": syntax error
2018-05-10 09:37:08  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 09:37:08] "GET /result HTTP/1.1" 500 -
2018-05-10 09:37:49  config.py[line:24] INFO 数据库初始化完成
2018-05-10 09:37:49  _internal.py[line:88] INFO  * Running on http://0.0.0.0:9090/ (Press CTRL+C to quit)
2018-05-10 09:37:54  config.py[line:24] INFO *************************收到访问请求: /*************************
2018-05-10 09:37:54  config.py[line:24] INFO 数据总量：20557
2018-05-10 09:37:54  config.py[line:24] INFO *************************请求处理完毕*************************
2018-05-10 09:37:54  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 09:37:54] "GET / HTTP/1.1" 200 -
2018-05-10 09:37:55  config.py[line:24] INFO *************************收到访问请求: /items/1*************************
2018-05-10 09:37:55  config.py[line:24] INFO 数据库分页查询sql: 

            SELECT * FROM items  ORDER BY good_id LIMIT 20 OFFSET 0 * 20
        
2018-05-10 09:37:55  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 09:37:55] "GET /items/1 HTTP/1.1" 200 -
2018-05-10 09:37:57  config.py[line:24] INFO *************************收到访问请求: /result*************************
2018-05-10 09:37:57  config.py[line:24] INFO 数据总量：0
2018-05-10 09:37:57  config.py[line:24] INFO *************************请求处理完毕*************************
2018-05-10 09:37:57  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 09:37:57] "GET /result HTTP/1.1" 200 -
2018-05-10 10:05:23  config.py[line:24] INFO 数据库初始化完成
2018-05-10 10:05:23  _internal.py[line:88] INFO  * Running on http://0.0.0.0:9090/ (Press CTRL+C to quit)
2018-05-10 10:05:26  config.py[line:24] INFO *************************收到访问请求: /*************************
2018-05-10 10:05:26  config.py[line:24] INFO 数据总量：20557
2018-05-10 10:05:26  config.py[line:24] INFO *************************请求处理完毕*************************
2018-05-10 10:05:27  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 10:05:27] "GET / HTTP/1.1" 200 -
2018-05-10 10:05:28  config.py[line:24] INFO *************************收到访问请求: /items/1*************************
2018-05-10 10:05:28  config.py[line:24] INFO 数据库分页查询sql: 

            SELECT * FROM items  ORDER BY good_id LIMIT 20 OFFSET 0 * 20
        
2018-05-10 10:05:28  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 10:05:28] "GET /items/1 HTTP/1.1" 200 -
2018-05-10 10:05:29  config.py[line:24] INFO *************************收到访问请求: /result*************************
2018-05-10 10:05:29  config.py[line:24] INFO 数据总量：2
2018-05-10 10:05:29  config.py[line:24] INFO *************************请求处理完毕*************************
2018-05-10 10:05:29  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 10:05:29] "GET /result HTTP/1.1" 200 -
2018-05-10 10:14:34  config.py[line:24] INFO 数据库初始化完成
2018-05-10 10:14:35  _internal.py[line:88] INFO  * Running on http://0.0.0.0:9090/ (Press CTRL+C to quit)
2018-05-10 10:14:37  config.py[line:24] INFO *************************收到访问请求: /*************************
2018-05-10 10:14:37  config.py[line:24] INFO 数据总量：20557
2018-05-10 10:14:37  config.py[line:24] INFO *************************请求处理完毕*************************
2018-05-10 10:14:37  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 10:14:37] "GET / HTTP/1.1" 200 -
2018-05-10 10:14:39  config.py[line:24] INFO *************************收到访问请求: /result*************************
2018-05-10 10:14:39  config.py[line:24] INFO 数据总量：20557
2018-05-10 10:14:39  config.py[line:24] INFO *************************请求处理完毕*************************
2018-05-10 10:14:39  app.py[line:1761] ERROR Exception on /result [GET]
Traceback (most recent call last):
  File "D:\python3\lib\site-packages\flask\app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\python3\lib\site-packages\flask\app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\python3\lib\site-packages\flask\app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "D:\python3\lib\site-packages\flask\_compat.py", line 35, in reraise
    raise value
  File "D:\python3\lib\site-packages\flask\app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\python3\lib\site-packages\flask\app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "D:/py_ws/PythonNote/PythonNote/webService/src/flask_main.py", line 67, in result
    return render_template('result.html', count=count_has_front)
  File "D:\python3\lib\site-packages\flask\templating.py", line 135, in render_template
    context, ctx.app)
  File "D:\python3\lib\site-packages\flask\templating.py", line 117, in _render
    rv = template.render(context)
  File "D:\python3\lib\site-packages\jinja2\asyncsupport.py", line 76, in render
    return original_render(self, *args, **kwargs)
  File "D:\python3\lib\site-packages\jinja2\environment.py", line 1008, in render
    return self.environment.handle_exception(exc_info, True)
  File "D:\python3\lib\site-packages\jinja2\environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "D:\python3\lib\site-packages\jinja2\_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "D:\py_ws\PythonNote\PythonNote\webService\src\templates\result.html", line 15, in top-level template code
    {{ index.render_comment(count) }}
jinja2.exceptions.UndefinedError: 'jinja2.environment.TemplateModule object' has no attribute 'render_comment'
2018-05-10 10:14:39  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 10:14:39] "GET /result HTTP/1.1" 500 -
2018-05-10 10:16:52  config.py[line:24] INFO 数据库初始化完成
2018-05-10 10:16:53  _internal.py[line:88] INFO  * Running on http://0.0.0.0:9090/ (Press CTRL+C to quit)
2018-05-10 10:16:56  config.py[line:24] INFO *************************收到访问请求: /*************************
2018-05-10 10:16:56  config.py[line:24] INFO 数据总量：20557
2018-05-10 10:16:56  config.py[line:24] INFO *************************请求处理完毕*************************
2018-05-10 10:16:56  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 10:16:56] "GET / HTTP/1.1" 200 -
2018-05-10 10:16:56  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 10:16:56] "GET /htmls/common.css HTTP/1.1" 200 -
2018-05-10 10:16:57  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 10:16:57] "GET /htmls/jquery-3.3.1.js HTTP/1.1" 200 -
2018-05-10 10:16:57  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 10:16:57] "GET /htmls/common.js HTTP/1.1" 200 -
2018-05-10 10:16:59  config.py[line:24] INFO *************************收到访问请求: /items/1*************************
2018-05-10 10:16:59  config.py[line:24] INFO 数据库分页查询sql: 

            SELECT * FROM items  ORDER BY good_id LIMIT 20 OFFSET 0 * 20
        
2018-05-10 10:16:59  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 10:16:59] "GET /items/1 HTTP/1.1" 200 -
2018-05-10 10:17:00  config.py[line:24] INFO *************************收到访问请求: /result*************************
2018-05-10 10:17:00  config.py[line:24] INFO 数据总量：20557
2018-05-10 10:17:00  config.py[line:24] INFO *************************请求处理完毕*************************
2018-05-10 10:17:00  app.py[line:1761] ERROR Exception on /result [GET]
Traceback (most recent call last):
  File "D:\python3\lib\site-packages\flask\app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\python3\lib\site-packages\flask\app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\python3\lib\site-packages\flask\app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "D:\python3\lib\site-packages\flask\_compat.py", line 35, in reraise
    raise value
  File "D:\python3\lib\site-packages\flask\app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\python3\lib\site-packages\flask\app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "D:/py_ws/PythonNote/PythonNote/webService/src/flask_main.py", line 67, in result
    return render_template('result.html', count=count_has_front)
  File "D:\python3\lib\site-packages\flask\templating.py", line 135, in render_template
    context, ctx.app)
  File "D:\python3\lib\site-packages\flask\templating.py", line 117, in _render
    rv = template.render(context)
  File "D:\python3\lib\site-packages\jinja2\asyncsupport.py", line 76, in render
    return original_render(self, *args, **kwargs)
  File "D:\python3\lib\site-packages\jinja2\environment.py", line 1008, in render
    return self.environment.handle_exception(exc_info, True)
  File "D:\python3\lib\site-packages\jinja2\environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "D:\python3\lib\site-packages\jinja2\_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "D:\py_ws\PythonNote\PythonNote\webService\src\templates\result.html", line 15, in top-level template code
    {{ index.render_comment(count) }}
jinja2.exceptions.UndefinedError: 'jinja2.environment.TemplateModule object' has no attribute 'render_comment'
2018-05-10 10:17:00  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 10:17:00] "GET /result HTTP/1.1" 500 -
2018-05-10 10:48:02  config.py[line:24] INFO 数据库初始化完成
2018-05-10 10:48:02  _internal.py[line:88] INFO  * Running on http://0.0.0.0:9090/ (Press CTRL+C to quit)
2018-05-10 10:48:28  config.py[line:24] INFO *************************收到访问请求: /*************************
2018-05-10 10:48:28  config.py[line:24] INFO 数据总量：20557
2018-05-10 10:48:28  config.py[line:24] INFO *************************请求处理完毕*************************
2018-05-10 10:48:28  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 10:48:28] "GET / HTTP/1.1" 200 -
2018-05-10 10:48:30  config.py[line:24] INFO *************************收到访问请求: /result*************************
2018-05-10 10:48:30  config.py[line:24] INFO 数据总量：20557
2018-05-10 10:48:30  config.py[line:24] INFO *************************请求处理完毕*************************
2018-05-10 10:48:30  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 10:48:30] "GET /result HTTP/1.1" 200 -
2018-05-10 11:00:07  config.py[line:24] INFO 数据库初始化完成
2018-05-10 11:00:07  _internal.py[line:88] INFO  * Running on http://0.0.0.0:9090/ (Press CTRL+C to quit)
2018-05-10 11:00:30  config.py[line:24] INFO *************************收到访问请求: /*************************
2018-05-10 11:00:30  config.py[line:24] INFO 数据总量：20557
2018-05-10 11:00:30  config.py[line:24] INFO *************************请求处理完毕*************************
2018-05-10 11:00:31  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 11:00:31] "GET / HTTP/1.1" 200 -
2018-05-10 11:00:33  config.py[line:24] INFO *************************收到访问请求: /result*************************
2018-05-10 11:00:33  config.py[line:24] INFO 数据总量：20557
2018-05-10 11:00:33  config.py[line:24] INFO *************************请求处理完毕*************************
2018-05-10 11:00:33  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 11:00:33] "GET /result HTTP/1.1" 200 -
2018-05-10 11:01:51  config.py[line:24] INFO 数据库初始化完成
2018-05-10 11:01:52  _internal.py[line:88] INFO  * Running on http://0.0.0.0:9090/ (Press CTRL+C to quit)
2018-05-10 11:01:56  config.py[line:24] INFO *************************收到访问请求: /*************************
2018-05-10 11:01:56  config.py[line:24] INFO 数据总量：20557
2018-05-10 11:01:56  config.py[line:24] INFO *************************请求处理完毕*************************
2018-05-10 11:01:56  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 11:01:56] "GET / HTTP/1.1" 200 -
2018-05-10 11:04:06  config.py[line:24] INFO 数据库初始化完成
2018-05-10 11:04:06  _internal.py[line:88] INFO  * Running on http://0.0.0.0:9090/ (Press CTRL+C to quit)
2018-05-10 11:04:10  config.py[line:24] INFO *************************收到访问请求: /*************************
2018-05-10 11:04:10  config.py[line:24] INFO 数据总量：20557
2018-05-10 11:04:10  config.py[line:24] INFO *************************请求处理完毕*************************
2018-05-10 11:04:10  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 11:04:10] "GET / HTTP/1.1" 200 -
2018-05-10 11:04:56  config.py[line:24] INFO 数据库初始化完成
2018-05-10 11:04:57  _internal.py[line:88] INFO  * Running on http://0.0.0.0:9090/ (Press CTRL+C to quit)
2018-05-10 11:04:59  config.py[line:24] INFO *************************收到访问请求: /*************************
2018-05-10 11:04:59  config.py[line:24] INFO 数据总量：20557
2018-05-10 11:04:59  config.py[line:24] INFO *************************请求处理完毕*************************
2018-05-10 11:04:59  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 11:04:59] "GET / HTTP/1.1" 200 -
2018-05-10 11:05:00  config.py[line:24] INFO *************************收到访问请求: /*************************
2018-05-10 11:05:00  config.py[line:24] INFO 数据总量：20557
2018-05-10 11:05:00  config.py[line:24] INFO *************************请求处理完毕*************************
2018-05-10 11:05:00  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 11:05:00] "GET / HTTP/1.1" 200 -
2018-05-10 11:05:00  config.py[line:24] INFO *************************收到访问请求: /*************************
2018-05-10 11:05:00  config.py[line:24] INFO 数据总量：20557
2018-05-10 11:05:00  config.py[line:24] INFO *************************请求处理完毕*************************
2018-05-10 11:05:00  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 11:05:00] "GET / HTTP/1.1" 200 -
2018-05-10 11:05:01  config.py[line:24] INFO *************************收到访问请求: /*************************
2018-05-10 11:05:01  config.py[line:24] INFO 数据总量：20557
2018-05-10 11:05:01  config.py[line:24] INFO *************************请求处理完毕*************************
2018-05-10 11:05:01  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 11:05:01] "GET / HTTP/1.1" 200 -
2018-05-10 11:05:01  config.py[line:24] INFO *************************收到访问请求: /*************************
2018-05-10 11:05:01  config.py[line:24] INFO 数据总量：20557
2018-05-10 11:05:01  config.py[line:24] INFO *************************请求处理完毕*************************
2018-05-10 11:05:01  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 11:05:01] "GET / HTTP/1.1" 200 -
2018-05-10 11:05:12  config.py[line:24] INFO *************************收到访问请求: /*************************
2018-05-10 11:05:12  config.py[line:24] INFO 数据总量：20557
2018-05-10 11:05:12  config.py[line:24] INFO *************************请求处理完毕*************************
2018-05-10 11:05:12  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 11:05:12] "GET / HTTP/1.1" 200 -
2018-05-10 11:05:12  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 11:05:12] "GET /htmls/common.css HTTP/1.1" 200 -
2018-05-10 11:05:12  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 11:05:12] "GET /htmls/jquery-3.3.1.js HTTP/1.1" 200 -
2018-05-10 11:05:12  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 11:05:12] "GET /htmls/common.js HTTP/1.1" 200 -
2018-05-10 11:05:13  config.py[line:24] INFO *************************收到访问请求: /*************************
2018-05-10 11:05:13  config.py[line:24] INFO 数据总量：20557
2018-05-10 11:05:13  config.py[line:24] INFO *************************请求处理完毕*************************
2018-05-10 11:05:13  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 11:05:13] "GET / HTTP/1.1" 200 -
2018-05-10 11:05:13  config.py[line:24] INFO *************************收到访问请求: /*************************
2018-05-10 11:05:13  config.py[line:24] INFO 数据总量：20557
2018-05-10 11:05:13  config.py[line:24] INFO *************************请求处理完毕*************************
2018-05-10 11:05:13  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 11:05:13] "GET / HTTP/1.1" 200 -
2018-05-10 11:05:14  config.py[line:24] INFO *************************收到访问请求: /*************************
2018-05-10 11:05:14  config.py[line:24] INFO 数据总量：20557
2018-05-10 11:05:14  config.py[line:24] INFO *************************请求处理完毕*************************
2018-05-10 11:05:14  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 11:05:14] "GET / HTTP/1.1" 200 -
2018-05-10 11:06:04  config.py[line:24] INFO 数据库初始化完成
2018-05-10 11:06:04  _internal.py[line:88] INFO  * Running on http://0.0.0.0:9090/ (Press CTRL+C to quit)
2018-05-10 11:06:06  config.py[line:24] INFO *************************收到访问请求: /*************************
2018-05-10 11:06:06  config.py[line:24] INFO 数据总量：20557
2018-05-10 11:06:06  config.py[line:24] INFO *************************请求处理完毕*************************
2018-05-10 11:06:06  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 11:06:06] "GET / HTTP/1.1" 200 -
2018-05-10 11:06:10  config.py[line:24] INFO *************************收到访问请求: /items/3*************************
2018-05-10 11:06:10  config.py[line:24] INFO 数据库分页查询sql: 

            SELECT * FROM items  ORDER BY good_id LIMIT 20 OFFSET 2 * 20
        
2018-05-10 11:06:10  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 11:06:10] "GET /items/3 HTTP/1.1" 200 -
2018-05-10 11:06:12  config.py[line:24] INFO *************************收到访问请求: /result*************************
2018-05-10 11:06:12  config.py[line:24] INFO 数据总量：20557
2018-05-10 11:06:12  config.py[line:24] INFO *************************请求处理完毕*************************
2018-05-10 11:06:12  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 11:06:12] "GET /result HTTP/1.1" 200 -
2018-05-10 17:32:00  config.py[line:24] INFO 数据库初始化完成
2018-05-10 17:32:00  _internal.py[line:88] INFO  * Running on http://0.0.0.0:9090/ (Press CTRL+C to quit)
2018-05-10 17:33:50  config.py[line:24] INFO *************************收到访问请求: /*************************
2018-05-10 17:33:50  config.py[line:24] INFO 数据总量：20557
2018-05-10 17:33:50  config.py[line:24] INFO *************************请求处理完毕*************************
2018-05-10 17:33:51  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 17:33:51] "GET / HTTP/1.1" 200 -
2018-05-10 17:33:51  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 17:33:51] "GET /favicon.ico HTTP/1.1" 404 -
2018-05-10 17:33:53  config.py[line:24] INFO *************************收到访问请求: /result*************************
2018-05-10 17:33:54  app.py[line:1761] ERROR Exception on /result [GET]
Traceback (most recent call last):
  File "D:\python3\lib\site-packages\flask\app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\python3\lib\site-packages\flask\app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\python3\lib\site-packages\flask\app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "D:\python3\lib\site-packages\flask\_compat.py", line 35, in reraise
    raise value
  File "D:\python3\lib\site-packages\flask\app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\python3\lib\site-packages\flask\app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "D:/py_ws/PythonNote/PythonNote/webService/src/flask_main.py", line 71, in result
    Log.info('数据总量：%d' % str(data))
TypeError: %d format: a number is required, not str
2018-05-10 17:33:54  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 17:33:54] "GET /result HTTP/1.1" 500 -
2018-05-10 17:34:33  config.py[line:24] INFO 数据库初始化完成
2018-05-10 17:34:34  _internal.py[line:88] INFO  * Running on http://0.0.0.0:9090/ (Press CTRL+C to quit)
2018-05-10 17:34:38  config.py[line:24] INFO *************************收到访问请求: /*************************
2018-05-10 17:34:38  config.py[line:24] INFO 数据总量：20557
2018-05-10 17:34:38  config.py[line:24] INFO *************************请求处理完毕*************************
2018-05-10 17:34:38  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 17:34:38] "GET / HTTP/1.1" 200 -
2018-05-10 17:34:41  config.py[line:24] INFO *************************收到访问请求: /result*************************
2018-05-10 17:34:41  config.py[line:24] INFO 数据总量：{'count_front': 2, 'count_side': 2, 'count_obverse': 2, 'count_all': 2, 'count_any': 2}
2018-05-10 17:34:41  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 17:34:41] "GET /result HTTP/1.1" 200 -
2018-05-10 17:36:39  config.py[line:24] INFO 数据库初始化完成
2018-05-10 17:36:39  _internal.py[line:88] INFO  * Running on http://0.0.0.0:9090/ (Press CTRL+C to quit)
2018-05-10 17:36:43  config.py[line:24] INFO *************************收到访问请求: /*************************
2018-05-10 17:36:43  config.py[line:24] INFO 数据总量：20557
2018-05-10 17:36:43  config.py[line:24] INFO *************************请求处理完毕*************************
2018-05-10 17:36:43  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 17:36:43] "GET / HTTP/1.1" 200 -
2018-05-10 17:36:44  config.py[line:24] INFO *************************收到访问请求: /result*************************
2018-05-10 17:36:45  config.py[line:24] INFO 数据总量：{'count_front': 0, 'count_side': 0, 'count_obverse': 0, 'count_all': 0, 'count_any': 0}
2018-05-10 17:36:45  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 17:36:45] "GET /result HTTP/1.1" 200 -
2018-05-10 17:45:00  config.py[line:24] INFO 数据库初始化完成
2018-05-10 17:45:01  _internal.py[line:88] INFO  * Running on http://0.0.0.0:9090/ (Press CTRL+C to quit)
2018-05-10 17:45:05  config.py[line:24] INFO *************************收到访问请求: /*************************
2018-05-10 17:45:05  config.py[line:24] INFO 数据总量：20557
2018-05-10 17:45:05  config.py[line:24] INFO *************************/ 请求处理完毕*************************
2018-05-10 17:45:05  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 17:45:05] "GET / HTTP/1.1" 200 -
2018-05-10 17:45:06  config.py[line:24] INFO *************************收到访问请求: /result*************************
2018-05-10 17:45:07  config.py[line:24] INFO 数据总量：{'count_front': 0, 'count_side': 0, 'count_obverse': 0, 'count_all': 0, 'count_any': 0}
2018-05-10 17:45:07  config.py[line:24] INFO *************************/result 请求处理完毕*************************
2018-05-10 17:45:07  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 17:45:07] "GET /result HTTP/1.1" 200 -
2018-05-10 17:48:14  config.py[line:24] INFO *************************收到访问请求: /result*************************
2018-05-10 17:48:14  config.py[line:24] INFO 数据总量：{'count_front': 0, 'count_side': 0, 'count_obverse': 0, 'count_all': 0, 'count_any': 0}
2018-05-10 17:48:14  config.py[line:24] INFO *************************/result 请求处理完毕*************************
2018-05-10 17:48:14  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 17:48:14] "GET /result HTTP/1.1" 200 -
2018-05-10 17:48:14  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 17:48:14] "GET /htmls/common.css HTTP/1.1" 200 -
2018-05-10 17:48:14  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 17:48:14] "GET /htmls/common.js HTTP/1.1" 200 -
2018-05-10 17:48:14  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 17:48:14] "GET /htmls/jquery-3.3.1.js HTTP/1.1" 200 -
2018-05-10 17:49:43  config.py[line:24] INFO 数据库初始化完成
2018-05-10 17:49:43  _internal.py[line:88] INFO  * Running on http://0.0.0.0:9090/ (Press CTRL+C to quit)
2018-05-10 17:49:49  config.py[line:24] INFO *************************收到访问请求: /*************************
2018-05-10 17:49:49  config.py[line:24] INFO 数据总量：20557
2018-05-10 17:49:49  config.py[line:24] INFO *************************/ 请求处理完毕*************************
2018-05-10 17:49:49  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 17:49:49] "GET / HTTP/1.1" 200 -
2018-05-10 17:49:50  config.py[line:24] INFO *************************收到访问请求: /result*************************
2018-05-10 17:49:50  config.py[line:24] INFO 数据总量：{'count_front': 0, 'count_side': 0, 'count_obverse': 0, 'count_all': 0, 'count_any': 0}
2018-05-10 17:49:50  config.py[line:24] INFO *************************/result 请求处理完毕*************************
2018-05-10 17:49:50  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 17:49:50] "GET /result HTTP/1.1" 200 -
2018-05-10 17:50:29  config.py[line:24] INFO *************************收到访问请求: /result*************************
2018-05-10 17:50:29  config.py[line:24] INFO 数据总量：{'count_front': 0, 'count_side': 0, 'count_obverse': 0, 'count_all': 0, 'count_any': 0}
2018-05-10 17:50:29  config.py[line:24] INFO *************************/result 请求处理完毕*************************
2018-05-10 17:50:29  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 17:50:29] "GET /result HTTP/1.1" 200 -
2018-05-10 17:50:30  config.py[line:24] INFO *************************收到访问请求: /result*************************
2018-05-10 17:50:30  config.py[line:24] INFO 数据总量：{'count_front': 0, 'count_side': 0, 'count_obverse': 0, 'count_all': 0, 'count_any': 0}
2018-05-10 17:50:30  config.py[line:24] INFO *************************/result 请求处理完毕*************************
2018-05-10 17:50:30  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 17:50:30] "GET /result HTTP/1.1" 200 -
2018-05-10 17:50:30  config.py[line:24] INFO *************************收到访问请求: /result*************************
2018-05-10 17:50:30  config.py[line:24] INFO *************************收到访问请求: /result*************************
2018-05-10 17:50:30  app.py[line:1761] ERROR Exception on /result [GET]
Traceback (most recent call last):
  File "D:\python3\lib\site-packages\flask\app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\python3\lib\site-packages\flask\app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\python3\lib\site-packages\flask\app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "D:\python3\lib\site-packages\flask\_compat.py", line 35, in reraise
    raise value
  File "D:\python3\lib\site-packages\flask\app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\python3\lib\site-packages\flask\app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "D:/py_ws/PythonNote/PythonNote/webService/src/flask_main.py", line 64, in result
    count_front = DBHelper.select_count(options=(''' status like '1__' ''',))           # 拥有正面图片的商品总数
  File "D:\py_ws\PythonNote\PythonNote\webService\src\db\dbhelper.py", line 162, in select_count
    res = cls.cursor.execute(select_sql)
sqlite3.ProgrammingError: Recursive use of cursors not allowed.
2018-05-10 17:50:30  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 17:50:30] "GET /result HTTP/1.1" 500 -
2018-05-10 17:50:30  config.py[line:24] INFO 数据总量：{'count_front': 0, 'count_side': 0, 'count_obverse': 0, 'count_all': 0, 'count_any': 0}
2018-05-10 17:50:30  config.py[line:24] INFO *************************/result 请求处理完毕*************************
2018-05-10 17:50:30  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 17:50:30] "GET /result HTTP/1.1" 200 -
2018-05-10 17:50:30  config.py[line:24] INFO *************************收到访问请求: /result*************************
2018-05-10 17:50:31  config.py[line:24] INFO 数据总量：{'count_front': 0, 'count_side': 0, 'count_obverse': 0, 'count_all': 0, 'count_any': 0}
2018-05-10 17:50:31  config.py[line:24] INFO *************************/result 请求处理完毕*************************
2018-05-10 17:50:31  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 17:50:31] "GET /result HTTP/1.1" 200 -
2018-05-10 17:50:31  config.py[line:24] INFO *************************收到访问请求: /result*************************
2018-05-10 17:50:31  config.py[line:24] INFO 数据总量：{'count_front': 0, 'count_side': 0, 'count_obverse': 0, 'count_all': 0, 'count_any': 0}
2018-05-10 17:50:31  config.py[line:24] INFO *************************/result 请求处理完毕*************************
2018-05-10 17:50:31  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 17:50:31] "GET /result HTTP/1.1" 200 -
2018-05-10 17:50:32  config.py[line:24] INFO *************************收到访问请求: /result*************************
2018-05-10 17:50:32  config.py[line:24] INFO 数据总量：{'count_front': 0, 'count_side': 0, 'count_obverse': 0, 'count_all': 0, 'count_any': 0}
2018-05-10 17:50:32  config.py[line:24] INFO *************************/result 请求处理完毕*************************
2018-05-10 17:50:32  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 17:50:32] "GET /result HTTP/1.1" 200 -
2018-05-10 17:50:32  config.py[line:24] INFO *************************收到访问请求: /result*************************
2018-05-10 17:50:32  config.py[line:24] INFO 数据总量：{'count_front': 0, 'count_side': 0, 'count_obverse': 0, 'count_all': 0, 'count_any': 0}
2018-05-10 17:50:32  config.py[line:24] INFO *************************/result 请求处理完毕*************************
2018-05-10 17:50:32  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 17:50:32] "GET /result HTTP/1.1" 200 -
2018-05-10 17:50:32  config.py[line:24] INFO *************************收到访问请求: /result*************************
2018-05-10 17:50:33  config.py[line:24] INFO *************************收到访问请求: /result*************************
2018-05-10 17:50:33  app.py[line:1761] ERROR Exception on /result [GET]
Traceback (most recent call last):
  File "D:\python3\lib\site-packages\flask\app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\python3\lib\site-packages\flask\app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\python3\lib\site-packages\flask\app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "D:\python3\lib\site-packages\flask\_compat.py", line 35, in reraise
    raise value
  File "D:\python3\lib\site-packages\flask\app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\python3\lib\site-packages\flask\app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "D:/py_ws/PythonNote/PythonNote/webService/src/flask_main.py", line 64, in result
    count_front = DBHelper.select_count(options=(''' status like '1__' ''',))           # 拥有正面图片的商品总数
  File "D:\py_ws\PythonNote\PythonNote\webService\src\db\dbhelper.py", line 162, in select_count
    res = cls.cursor.execute(select_sql)
sqlite3.ProgrammingError: Recursive use of cursors not allowed.
2018-05-10 17:50:33  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 17:50:33] "GET /result HTTP/1.1" 500 -
2018-05-10 17:50:33  config.py[line:24] INFO 数据总量：{'count_front': 0, 'count_side': 0, 'count_obverse': 0, 'count_all': 0, 'count_any': 0}
2018-05-10 17:50:33  config.py[line:24] INFO *************************/result 请求处理完毕*************************
2018-05-10 17:50:33  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 17:50:33] "GET /result HTTP/1.1" 200 -
2018-05-10 17:50:33  config.py[line:24] INFO *************************收到访问请求: /result*************************
2018-05-10 17:50:33  config.py[line:24] INFO 数据总量：{'count_front': 0, 'count_side': 0, 'count_obverse': 0, 'count_all': 0, 'count_any': 0}
2018-05-10 17:50:33  config.py[line:24] INFO *************************/result 请求处理完毕*************************
2018-05-10 17:50:33  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 17:50:33] "GET /result HTTP/1.1" 200 -
2018-05-10 17:50:42  config.py[line:24] INFO *************************收到访问请求: /result*************************
2018-05-10 17:50:42  config.py[line:24] INFO 数据总量：{'count_front': 0, 'count_side': 0, 'count_obverse': 0, 'count_all': 0, 'count_any': 0}
2018-05-10 17:50:42  config.py[line:24] INFO *************************/result 请求处理完毕*************************
2018-05-10 17:50:42  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 17:50:42] "GET /result HTTP/1.1" 200 -
2018-05-10 17:50:42  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 17:50:42] "GET /htmls/jquery-3.3.1.js HTTP/1.1" 200 -
2018-05-10 17:50:42  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 17:50:42] "GET /htmls/common.css HTTP/1.1" 200 -
2018-05-10 17:50:42  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 17:50:42] "GET /htmls/common.js HTTP/1.1" 200 -
2018-05-10 17:50:59  config.py[line:24] INFO *************************收到访问请求: /result*************************
2018-05-10 17:50:59  config.py[line:24] INFO 数据总量：{'count_front': 0, 'count_side': 0, 'count_obverse': 0, 'count_all': 0, 'count_any': 0}
2018-05-10 17:50:59  config.py[line:24] INFO *************************/result 请求处理完毕*************************
2018-05-10 17:50:59  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 17:50:59] "GET /result HTTP/1.1" 200 -
2018-05-10 17:51:07  config.py[line:24] INFO *************************收到访问请求: /result*************************
2018-05-10 17:51:07  config.py[line:24] INFO 数据总量：{'count_front': 0, 'count_side': 0, 'count_obverse': 0, 'count_all': 0, 'count_any': 0}
2018-05-10 17:51:07  config.py[line:24] INFO *************************/result 请求处理完毕*************************
2018-05-10 17:51:07  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 17:51:07] "GET /result HTTP/1.1" 200 -
2018-05-10 17:51:07  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 17:51:07] "GET /htmls/common.css HTTP/1.1" 200 -
2018-05-10 17:51:07  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 17:51:07] "GET /htmls/jquery-3.3.1.js HTTP/1.1" 200 -
2018-05-10 17:51:07  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 17:51:07] "GET /htmls/common.js HTTP/1.1" 200 -
2018-05-10 17:51:08  config.py[line:24] INFO *************************收到访问请求: /result*************************
2018-05-10 17:51:08  config.py[line:24] INFO 数据总量：{'count_front': 0, 'count_side': 0, 'count_obverse': 0, 'count_all': 0, 'count_any': 0}
2018-05-10 17:51:08  config.py[line:24] INFO *************************/result 请求处理完毕*************************
2018-05-10 17:51:08  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 17:51:08] "GET /result HTTP/1.1" 200 -
2018-05-10 17:51:09  config.py[line:24] INFO *************************收到访问请求: /result*************************
2018-05-10 17:51:09  config.py[line:24] INFO 数据总量：{'count_front': 0, 'count_side': 0, 'count_obverse': 0, 'count_all': 0, 'count_any': 0}
2018-05-10 17:51:09  config.py[line:24] INFO *************************/result 请求处理完毕*************************
2018-05-10 17:51:09  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 17:51:09] "GET /result HTTP/1.1" 200 -
2018-05-10 17:51:44  config.py[line:24] INFO 数据库初始化完成
2018-05-10 17:51:44  _internal.py[line:88] INFO  * Running on http://0.0.0.0:9090/ (Press CTRL+C to quit)
2018-05-10 17:51:48  config.py[line:24] INFO *************************收到访问请求: /*************************
2018-05-10 17:51:48  config.py[line:24] INFO 数据总量：20557
2018-05-10 17:51:48  config.py[line:24] INFO *************************/ 请求处理完毕*************************
2018-05-10 17:51:48  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 17:51:48] "GET / HTTP/1.1" 200 -
2018-05-10 17:51:48  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 17:51:48] "GET /htmls/common.css HTTP/1.1" 200 -
2018-05-10 17:51:48  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 17:51:48] "GET /htmls/jquery-3.3.1.js HTTP/1.1" 200 -
2018-05-10 17:51:48  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 17:51:48] "GET /htmls/common.js HTTP/1.1" 200 -
2018-05-10 17:51:50  config.py[line:24] INFO *************************收到访问请求: /result*************************
2018-05-10 17:51:50  config.py[line:24] INFO 数据总量：{'count_front': 0, 'count_side': 0, 'count_obverse': 0, 'count_all': 0, 'count_any': 0}
2018-05-10 17:51:50  config.py[line:24] INFO *************************/result 请求处理完毕*************************
2018-05-10 17:51:50  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 17:51:50] "GET /result HTTP/1.1" 200 -
2018-05-10 17:52:31  config.py[line:24] INFO *************************收到访问请求: /result*************************
2018-05-10 17:52:31  config.py[line:24] INFO 数据总量：{'count_front': 0, 'count_side': 0, 'count_obverse': 0, 'count_all': 0, 'count_any': 0}
2018-05-10 17:52:31  config.py[line:24] INFO *************************/result 请求处理完毕*************************
2018-05-10 17:52:31  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 17:52:31] "GET /result HTTP/1.1" 200 -
2018-05-10 17:52:48  config.py[line:24] INFO *************************收到访问请求: /result*************************
2018-05-10 17:52:48  config.py[line:24] INFO 数据总量：{'count_front': 0, 'count_side': 0, 'count_obverse': 0, 'count_all': 0, 'count_any': 0}
2018-05-10 17:52:48  config.py[line:24] INFO *************************/result 请求处理完毕*************************
2018-05-10 17:52:48  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 17:52:48] "GET /result HTTP/1.1" 200 -
2018-05-10 17:52:48  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 17:52:48] "GET /htmls/jquery-3.3.1.js HTTP/1.1" 200 -
2018-05-10 17:52:48  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 17:52:48] "GET /htmls/common.css HTTP/1.1" 200 -
2018-05-10 17:52:48  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 17:52:48] "GET /htmls/common.js HTTP/1.1" 200 -
2018-05-10 17:52:48  config.py[line:24] INFO *************************收到访问请求: /result*************************
2018-05-10 17:52:48  config.py[line:24] INFO 数据总量：{'count_front': 0, 'count_side': 0, 'count_obverse': 0, 'count_all': 0, 'count_any': 0}
2018-05-10 17:52:48  config.py[line:24] INFO *************************/result 请求处理完毕*************************
2018-05-10 17:52:48  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 17:52:48] "GET /result HTTP/1.1" 200 -
2018-05-10 17:52:48  config.py[line:24] INFO *************************收到访问请求: /result*************************
2018-05-10 17:52:48  config.py[line:24] INFO 数据总量：{'count_front': 0, 'count_side': 0, 'count_obverse': 0, 'count_all': 0, 'count_any': 0}
2018-05-10 17:52:48  config.py[line:24] INFO *************************/result 请求处理完毕*************************
2018-05-10 17:52:48  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 17:52:48] "GET /result HTTP/1.1" 200 -
2018-05-10 17:53:03  config.py[line:24] INFO 数据库初始化完成
2018-05-10 17:53:03  _internal.py[line:88] INFO  * Running on http://0.0.0.0:9090/ (Press CTRL+C to quit)
2018-05-10 17:53:06  config.py[line:24] INFO *************************收到访问请求: /result*************************
2018-05-10 17:53:07  config.py[line:24] INFO 数据总量：{'count_front': 0, 'count_side': 0, 'count_obverse': 0, 'count_all': 0, 'count_any': 0}
2018-05-10 17:53:07  config.py[line:24] INFO *************************/result 请求处理完毕*************************
2018-05-10 17:53:07  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 17:53:07] "GET /result HTTP/1.1" 200 -
2018-05-10 17:58:25  config.py[line:24] INFO 数据库初始化完成
2018-05-10 17:58:25  _internal.py[line:88] INFO  * Running on http://0.0.0.0:9090/ (Press CTRL+C to quit)
2018-05-10 17:58:29  config.py[line:24] INFO *************************收到访问请求: /result*************************
2018-05-10 17:58:29  config.py[line:24] INFO 数据总量：{'count_front': 0, 'count_side': 0, 'count_obverse': 0, 'count_all': 0, 'count_any': 0}
2018-05-10 17:58:29  config.py[line:24] INFO *************************/result 请求处理完毕*************************
2018-05-10 17:58:29  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 17:58:29] "GET /result HTTP/1.1" 200 -
2018-05-10 17:58:57  config.py[line:24] INFO *************************收到访问请求: /result*************************
2018-05-10 17:58:57  config.py[line:24] INFO 数据总量：{'count_front': 0, 'count_side': 0, 'count_obverse': 0, 'count_all': 0, 'count_any': 0}
2018-05-10 17:58:57  config.py[line:24] INFO *************************/result 请求处理完毕*************************
2018-05-10 17:58:57  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 17:58:57] "GET /result HTTP/1.1" 200 -
2018-05-10 17:58:57  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 17:58:57] "GET /htmls/jquery-3.3.1.js HTTP/1.1" 200 -
2018-05-10 17:58:57  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 17:58:57] "GET /htmls/common.css HTTP/1.1" 200 -
2018-05-10 17:58:57  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 17:58:57] "GET /htmls/common.js HTTP/1.1" 200 -
2018-05-10 17:58:59  config.py[line:24] INFO *************************收到访问请求: /result*************************
2018-05-10 17:58:59  config.py[line:24] INFO 数据总量：{'count_front': 0, 'count_side': 0, 'count_obverse': 0, 'count_all': 0, 'count_any': 0}
2018-05-10 17:58:59  config.py[line:24] INFO *************************/result 请求处理完毕*************************
2018-05-10 17:58:59  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 17:58:59] "GET /result HTTP/1.1" 200 -
2018-05-10 18:10:00  config.py[line:24] INFO 数据库初始化完成
2018-05-10 18:10:00  _internal.py[line:88] INFO  * Running on http://0.0.0.0:9090/ (Press CTRL+C to quit)
2018-05-10 18:10:02  config.py[line:24] INFO *************************收到访问请求: /result*************************
2018-05-10 18:10:03  config.py[line:24] INFO 数据总量：{'count_front': 0, 'count_side': 0, 'count_obverse': 0, 'count_all': 0, 'count_any': 0, 'pages_all': []}
2018-05-10 18:10:03  config.py[line:24] INFO *************************/result 请求处理完毕*************************
2018-05-10 18:10:03  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 18:10:03] "GET /result HTTP/1.1" 200 -
2018-05-10 18:10:05  config.py[line:24] INFO *************************收到访问请求: /result*************************
2018-05-10 18:10:05  config.py[line:24] INFO 数据总量：{'count_front': 0, 'count_side': 0, 'count_obverse': 0, 'count_all': 0, 'count_any': 0, 'pages_all': []}
2018-05-10 18:10:05  config.py[line:24] INFO *************************/result 请求处理完毕*************************
2018-05-10 18:10:05  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 18:10:05] "GET /result HTTP/1.1" 200 -
2018-05-10 18:10:17  config.py[line:24] INFO *************************收到访问请求: /result*************************
2018-05-10 18:10:18  config.py[line:24] INFO 数据总量：{'count_front': 0, 'count_side': 0, 'count_obverse': 0, 'count_all': 0, 'count_any': 0, 'pages_all': []}
2018-05-10 18:10:18  config.py[line:24] INFO *************************/result 请求处理完毕*************************
2018-05-10 18:10:18  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 18:10:18] "GET /result HTTP/1.1" 200 -
2018-05-10 18:10:18  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 18:10:18] "GET /htmls/jquery-3.3.1.js HTTP/1.1" 200 -
2018-05-10 18:10:18  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 18:10:18] "GET /htmls/common.css HTTP/1.1" 200 -
2018-05-10 18:10:18  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 18:10:18] "GET /htmls/common.js HTTP/1.1" 200 -
2018-05-10 18:10:18  config.py[line:24] INFO *************************收到访问请求: /result*************************
2018-05-10 18:10:19  config.py[line:24] INFO 数据总量：{'count_front': 0, 'count_side': 0, 'count_obverse': 0, 'count_all': 0, 'count_any': 0, 'pages_all': []}
2018-05-10 18:10:19  config.py[line:24] INFO *************************/result 请求处理完毕*************************
2018-05-10 18:10:19  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 18:10:19] "GET /result HTTP/1.1" 200 -
2018-05-10 18:10:19  config.py[line:24] INFO *************************收到访问请求: /result*************************
2018-05-10 18:10:19  config.py[line:24] INFO 数据总量：{'count_front': 0, 'count_side': 0, 'count_obverse': 0, 'count_all': 0, 'count_any': 0, 'pages_all': []}
2018-05-10 18:10:19  config.py[line:24] INFO *************************/result 请求处理完毕*************************
2018-05-10 18:10:19  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 18:10:19] "GET /result HTTP/1.1" 200 -
2018-05-10 18:10:19  config.py[line:24] INFO *************************收到访问请求: /result*************************
2018-05-10 18:10:19  config.py[line:24] INFO 数据总量：{'count_front': 0, 'count_side': 0, 'count_obverse': 0, 'count_all': 0, 'count_any': 0, 'pages_all': []}
2018-05-10 18:10:19  config.py[line:24] INFO *************************/result 请求处理完毕*************************
2018-05-10 18:10:19  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 18:10:19] "GET /result HTTP/1.1" 200 -
2018-05-10 18:10:19  config.py[line:24] INFO *************************收到访问请求: /result*************************
2018-05-10 18:10:19  config.py[line:24] INFO 数据总量：{'count_front': 0, 'count_side': 0, 'count_obverse': 0, 'count_all': 0, 'count_any': 0, 'pages_all': []}
2018-05-10 18:10:19  config.py[line:24] INFO *************************/result 请求处理完毕*************************
2018-05-10 18:10:19  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 18:10:19] "GET /result HTTP/1.1" 200 -
2018-05-10 18:10:19  config.py[line:24] INFO *************************收到访问请求: /result*************************
2018-05-10 18:10:20  config.py[line:24] INFO 数据总量：{'count_front': 0, 'count_side': 0, 'count_obverse': 0, 'count_all': 0, 'count_any': 0, 'pages_all': []}
2018-05-10 18:10:20  config.py[line:24] INFO *************************/result 请求处理完毕*************************
2018-05-10 18:10:20  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 18:10:20] "GET /result HTTP/1.1" 200 -
2018-05-10 18:10:20  config.py[line:24] INFO *************************收到访问请求: /result*************************
2018-05-10 18:10:20  config.py[line:24] INFO 数据总量：{'count_front': 0, 'count_side': 0, 'count_obverse': 0, 'count_all': 0, 'count_any': 0, 'pages_all': []}
2018-05-10 18:10:20  config.py[line:24] INFO *************************/result 请求处理完毕*************************
2018-05-10 18:10:20  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 18:10:20] "GET /result HTTP/1.1" 200 -
2018-05-10 18:13:26  config.py[line:24] INFO 数据库初始化完成
2018-05-10 18:13:27  _internal.py[line:88] INFO  * Running on http://0.0.0.0:9090/ (Press CTRL+C to quit)
2018-05-10 18:13:29  config.py[line:24] INFO *************************收到访问请求: /result*************************
2018-05-10 18:13:29  config.py[line:24] INFO 数据总量：{'count_front': 0, 'count_side': 0, 'count_obverse': 0, 'count_all': 0, 'count_any': 0, 'pages_all': []}
2018-05-10 18:13:29  config.py[line:24] INFO *************************/result 请求处理完毕*************************
2018-05-10 18:13:29  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 18:13:29] "GET /result HTTP/1.1" 200 -
2018-05-10 18:13:30  config.py[line:24] INFO *************************收到访问请求: /result*************************
2018-05-10 18:13:30  config.py[line:24] INFO 数据总量：{'count_front': 0, 'count_side': 0, 'count_obverse': 0, 'count_all': 0, 'count_any': 0, 'pages_all': []}
2018-05-10 18:13:30  config.py[line:24] INFO *************************/result 请求处理完毕*************************
2018-05-10 18:13:30  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 18:13:30] "GET /result HTTP/1.1" 200 -
2018-05-10 18:13:30  config.py[line:24] INFO *************************收到访问请求: /result*************************
2018-05-10 18:13:31  config.py[line:24] INFO 数据总量：{'count_front': 0, 'count_side': 0, 'count_obverse': 0, 'count_all': 0, 'count_any': 0, 'pages_all': []}
2018-05-10 18:13:31  config.py[line:24] INFO *************************/result 请求处理完毕*************************
2018-05-10 18:13:31  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 18:13:31] "GET /result HTTP/1.1" 200 -
2018-05-10 18:13:31  config.py[line:24] INFO *************************收到访问请求: /result*************************
2018-05-10 18:13:31  config.py[line:24] INFO 数据总量：{'count_front': 0, 'count_side': 0, 'count_obverse': 0, 'count_all': 0, 'count_any': 0, 'pages_all': []}
2018-05-10 18:13:31  config.py[line:24] INFO *************************/result 请求处理完毕*************************
2018-05-10 18:13:31  _internal.py[line:88] INFO 127.0.0.1 - - [10/May/2018 18:13:31] "GET /result HTTP/1.1" 200 -
